[2021-06-29 13:15:51,058] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: open_aq.store_data 2021-06-29T13:15:33.909199+00:00 [queued]>
[2021-06-29 13:15:51,110] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: open_aq.store_data 2021-06-29T13:15:33.909199+00:00 [queued]>
[2021-06-29 13:15:51,110] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-29 13:15:51,110] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-06-29 13:15:51,111] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-29 13:15:51,139] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): store_data> on 2021-06-29T13:15:33.909199+00:00
[2021-06-29 13:15:51,154] {standard_task_runner.py:52} INFO - Started process 14445 to run task
[2021-06-29 13:15:51,183] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'open_aq', 'store_data', '2021-06-29T13:15:33.909199+00:00', '--job-id', '539', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/open_aq.py', '--cfg-path', '/tmp/tmpfsc_pzcs', '--error-file', '/tmp/tmpzi7qksfw']
[2021-06-29 13:15:51,184] {standard_task_runner.py:77} INFO - Job 539: Subtask store_data
[2021-06-29 13:15:51,351] {logging_mixin.py:104} INFO - Running <TaskInstance: open_aq.store_data 2021-06-29T13:15:33.909199+00:00 [running]> on host airflowvm
[2021-06-29 13:15:51,505] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=open_aq
AIRFLOW_CTX_TASK_ID=store_data
AIRFLOW_CTX_EXECUTION_DATE=2021-06-29T13:15:33.909199+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-06-29T13:15:33.909199+00:00
[2021-06-29 13:15:51,543] {base.py:69} INFO - Using connection to: id: ***_default. Host: localhost, Port: 5432, Schema: , Login: ***, Password: ***, extra: {'cursor': 'realdictcursor'}
[2021-06-29 13:15:51,565] {python.py:151} INFO - Done. Returned value was: None
[2021-06-29 13:15:51,602] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=open_aq, task_id=store_data, execution_date=20210629T131533, start_date=20210629T131551, end_date=20210629T131551
[2021-06-29 13:15:51,708] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-06-29 13:15:51,754] {local_task_job.py:151} INFO - Task exited with return code 0
